packages/amplify-category-geo/src/service-walkthroughs/resourceWalkthrough.ts

```diff
@@ -29,22 +30,28 @@ export async function pricingPlanWalkthrough<T extends ResourceParameters>(
 ): Promise<Partial<T>> {
     let pricingPlan: PricingPlan = parameters.pricingPlan ? parameters.pricingPlan : PricingPlan.RequestBasedUsage;
 
-    context.print.info(choosePricingPlan);
+    printer.info(choosePricingPlan);
 
     const pricingPlanBusinessTypeChoices = [
-        { name: 'No, I only need to track consumers personal mobile devices', value: false },
-        { name: 'Yes, I track commercial assets (For example, any mobile object that is tracked by a company in support of its business)', value: true }
+        { name: "No, I do not track devices or I only need to track consumers' personal devices", value: PricingPlan.RequestBasedUsage },
+        { name: 'Yes, I track commercial assets (For example, any mobile object that is tracked by a company in support of its business)', value: 'Unknown' },
+        { name: 'Learn More', value: 'LearnMore'}
     ];
     const pricingPlanBusinessTypePrompt = {
         type: 'list',
         name: 'pricingPlanBusinessType',
         message: 'Are you tracking commercial assets for your business in your app?',
         choices: pricingPlanBusinessTypeChoices,
-        default: pricingPlan === PricingPlan.RequestBasedUsage ? false : true
+        default: pricingPlan === PricingPlan.RequestBasedUsage ? PricingPlan.RequestBasedUsage : 'Unknown'
     };
 
-    const pricingPlanBusinessTypeChoice = ((await inquirer.prompt([pricingPlanBusinessTypePrompt])).pricingPlanBusinessType as boolean);
-    if (pricingPlanBusinessTypeChoice === false) {
+    let pricingPlanBusinessTypeChoice = ((await inquirer.prompt([pricingPlanBusinessTypePrompt])).pricingPlanBusinessType);
```

Comment:
would be nice to migrate to `prompter.pick()` for this prompt and the ones below. Should be something like:
```suggestion
    let pricingPlanBusinessTypeChoice = await prompter.pick('Are you tracking commercial...', pricingPlanBusinessTypeChoices)
```

Reply:
I did try to adapt the new `prompter` and faced some issues with `pick`.

Without using `prompter.pick`:

![Screen Shot 2021-09-13 at 9 54 16 AM](https://user-images.githubusercontent.com/55896475/133125669-89fc51b0-63ff-4c9a-ab87-1d49e72c2005.png)
And when I hit cancel(`ctrl+z`), it graciously terminates
![Screen Shot 2021-09-13 at 9 54 44 AM](https://user-images.githubusercontent.com/55896475/133125757-32af1ab1-b3b4-4445-9bf0-4bf6ea5c98c9.png)

Using `prompter.pick`:
![Screen Shot 2021-09-13 at 9 51 02 AM](https://user-images.githubusercontent.com/55896475/133125807-770d7140-4ec4-4762-85df-2c266f26a5ea.png)
The cursor is at the wrong place which causes the terminal to be left in weird state when we hit `ctrl+z`. 
![Screen Shot 2021-09-13 at 9 51 30 AM](https://user-images.githubusercontent.com/55896475/133126009-c6074128-80af-48ef-b8d7-d2f029b783ae.png)
Moreover, the options displayed are cleaner with `>` in front of them and a message `use arrow keys`. 

Another DX related issue. When we use array of name-value pairs in pick, we need to specify the index as default. This might need additional effort to compute the index as currently we specify the value as default, not the index. 

I would like to come back to this in a separate PR when these issues are resolved.

Reply:
What's your use case for stopping the process? I tried out stopping and resuming the process when in the middle of an inquirer prompt and there's an error there as well. If you just want to exit the process, ctrl+c will work.

Re: the `Use arrow keys` suggestion, yes that's good feedback, we can add that. However, I would still recommend using prompter.pick() and when that update lands you'll get it for free

As for specifying the index rather than the value of the initial selection, this was an intentional choice because if the type of the values in the list are numbers it's ambiguous if you intend to select a value or an index. Furthermore, since the list value type can be anything, it may not be sufficient to compare values using `===` Since an array of values must already be provided, it should be trivial to pass in an initial index of `selectionList.indexOf(defaultValue)`

Reply:
The use-case for stopping the current flow is when I am not sure of the options to choose and might want to add the resource at a later time. The process is stopped successfully but the output is not cleared graciously as you can see from the screenshot.

Reply:
@phani-srikar  Right, but as far as I can tell, the current prompts that use inquirer don't support resuming the process in the middle of a prompt either. If you want to stop the process and restart it later, why not use ctrl+c?